frontend/test/metabase/scenarios/collections/collection-pinned-overview.cy.spec.js

```diff
@@ -135,6 +152,51 @@ describe("scenarios > collection pinned items overview", () => {
     getPinnedSection().should("not.exist");
     cy.findByText(DASHBOARD_NAME).should("not.exist");
   });
+
+  it("should be able to hide the visualization for a pinned question", () => {
+    cy.request("PUT", "/api/card/2", { collection_position: 1 });
+
+    openRootCollection();
+    openPinnedItemMenu(QUESTION_NAME);
+    popover().within(() => cy.findByText("Don’t show visualization").click());
```

Comment:
nit: when you only want to interact with a single element in some container, you can skip the `within` part:

```js
popover().findByText("Don't show visualization").click()
```

Reply:
I didn't know that. Thanks!

Reply:
Fixed